spec.md

```diff
@@ -843,6 +853,13 @@ message CreateVolumeRequest {
   // VOLUME_ACCESSIBILITY_CONSTRAINTS plugin capability, the SP MAY
   // choose where the provisioned volume is accessible from.
   TopologyRequirement accessibility_requirements = 7;
+
+  // Plugin specific mutable parameters passed in as opaque key-value
+  // pairs. This field is OPTIONAL. The Plugin is responsible for 
+  // parsing and validating these parameters. COs will treat these 
+  // as opaque. Plugins SHOULD treat these as if they take precedence 
+  // over the parameters field.
+  map<string, string> mutable_parameters = 8 [(alpha_field) = true];
```

Comment:
perhaps a note that indicates this mutable parameters field SHALL be ignored by the plugin if MUTABLE_VOLUME controller capability is not supported.

Reply:
I agree with this but, I actually prefer the reverse of the language. We should say that the CO SHALL NOT provide anything in that field if the capability isn't advertised. We could even make both statements for double protection.

Reply:
I added:

```
// COs SHALL NOT provide any values in mutable_parameters if the
// capability is not enabled.
```